R/cfr.R

```diff
@@ -263,9 +251,17 @@ cfr_server <- function(
                 name = estimate_label(),
                 color = "darkred"
               ) %>%
-              highcharter::hc_tooltip(shared = TRUE, sort = TRUE)
+              highcharter::hc_chart(zoomType = "xy") %>% 
+              highcharter::hc_xAxis(type = "datetime") %>% 
+              highcharter::hc_yAxis_multiples(
+                list(title = list(text = ""), max = 1),
+                list(title = list(text = ""), opposite = TRUE, linkedTo = 0)
+              ) %>%
+              highcharter::hc_plotOptions(line = list(marker = list(enabled = FALSE))) %>% 
+              highcharter::hc_tooltip(shared = TRUE, sort = TRUE, valueDecimals = 3) %>% 
+              my_hc_export()
           ),
-          input$go
+          cfr_estimate()
```

Comment:
Am I correct in understanding that binding to events needs to be specified as a cascade? So the highcharter reactive is bound to `cfr_estimate()` rather than the action button - and the binding to the action button is inherited?

Reply:
Yes exactly. You don't want the plot to be re-triggered by the action button because it depends on `cfr_estimate()` and this may take some time to be calculated after the button is clicked. So binding the plot to changes in `cfr_estimate()` which itself is bound to `input$go` is the correct flow of reactive events.